src/components/device/ipbase.ts

```diff
@@ -108,10 +107,10 @@ export default class IpBaseDevice implements IIpDeviceManager, IUVCControls {
    * Grpc client instance for communicating with the grpc server running on the network device
    *
    * @private
-   * @type {HuddlyServiceClient}
+   * @type {HuddlyServiceClient | HuddlyGrpcTunnelClient}
    * @memberof IpBaseDevice
    */
-  private huddlyGrpcClient: HuddlyServiceClient;
+  private huddlyGrpcClient: HuddlyServiceClient | HuddlyGrpcTunnelClient;
```

Comment:
I think we should consider if we can drop the `HuddlyGrpcTunnelClient` interface and just use the `HuddlyServiceClient` for the tunnel as well. Yes it will have lots of empty functions, but it avoids polluting these ipbase classes with tunnel stuff.

Reply:
Yes, I did consider this, but just an empty implementation leaves us with a file of almost 1500 lines. Bit too much richness to have on display imo. We'd also have to deal with updating it as we add new version of the grpc api which will become a bit cumbersome. 
What about a mention in the docstring so that people understand its purpose?